### PR TITLE
alpha 2 fixes

### PR DESCRIPTION
**Pull request by Aprillion**

### CODE REVIEW

File: app/components/question.tsx [hunk 1]
```diff
@@ -35,6 +34,31 @@ export default function Question({
...
+        hideEl?.addEventListener('click', hideLong)
+
+        return () => {
+          showEl.removeEventListener('click', showLong)
+          hideEl?.removeEventListener('click', hideLong)
+        }
+      }
+    }
+  }, [isExpandedAfterLoading])
+
```
Comment (ccstan99) [inline app/components/question.tsx:61]: Is this something that's marked in the wiki with short/long description?

**Review by ccstan99 — APPROVED** [review]
Looks good. FYI, this tooltip shows up when the search box is empty.
Screen Shot 2022-07-01 at 11 44 19 AM

File: app/components/question.tsx (hunk 1 above)
Comment (Aprillion) [reply app/components/question.tsx:61]: yup, e.g. https://stampy.ai/wiki/How_do_I_format_answers_on_Stampy%3F => https://ui.stampy.ai/?state=7503_

Comment (Aprillion) [reply app/components/question.tsx:61]: I would be more happy with just 1 length of each answer, but @plexish is still confident this is a useful feature ¯\\\_(ツ)_/¯

**Comment by Aprillion** [comment]
> Looks good. FYI, this tooltip shows up when the search box is empty.

hm.. I don't see it in the HTML or in my browser .. could it be from some browser extension? if it happens with some particular repro steps, please open a new issue and we can investigate later

File: app/components/question.tsx (hunk 1 above)
Comment (ccstan99) [reply app/components/question.tsx:61]: Yes, I personally believe it's a healthy exercise to give concise answers in an FAQ. If an answer is really long, it probably really should just be broken up into related questions/answers.

**Pull request merged**